app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
```

Comment:
fair point, ktory samozrejme dava zmysel. skoda, ze si nevidel prvu verziu harvestora, ktory bol kompozicne pouzitelny na vsetky endpointy. po naslednych reviews som vsak pochopil, ze akakolvek abstrakcia, respektive definovanie dedicnosti je unahleny krok. je mi samozrejme jasne, ze by kazdy harvester stratil na kode, vyzeral aspon o trochu jednoduchsie, ale za aku cenu? za cenu dalsich testov, ktore ma budu stat cas. za cenu, ak by sa kompozicia v buducnosti ukazala ako lepsie riesenie, cela dedicnost by sa musela odstranovat. nehovoriac, ze uz len tymto krokom urcujes buduci design, o ktorom aktualne niesom presvedceny. suma sumarov, nepride mi to zmena, ktora sa v buducnosti vyplati. ak mas preargumenty, sem s nimi, rad si ich vypocujem :)